pkgs/tools/networking/httping/default.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation rec {
     sha256 = "1y7sbgkhgadmd93x1zafqc4yp26ssiv16ni5bbi9vmvvdl55m29y";
   };
 
-  buildInputs = [ fftw ncurses openssl ];
+  buildInputs = [ fftw ncurses openssl ] ++ (with stdenv; lib.optional isDarwin libintl);
```

Comment:
Since libintl is null on Glibc Linux, you don't need the lib.optional part. It can just be listed a build inputs. As a plus, this makes your changes compatible with other libcs that don't come with libintl like Musl!